src/components/language-toggle/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@govuk-one-login/one-login-language-toggle",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
Why have we put "2" for this new release?

Reply:
I went for a major version change as the new parameter of visuallyHidden needs to be added to the initialisation of the component... thinking about it now, i'm not sure if it will break anything as it's a parameter that's optional